website/pages/en/about.mdx

```diff
@@ -44,4 +44,4 @@ The flow follows these steps:
 
 In the following sections we will go into more detail on how to define subgraphs, how to deploy them, and how to query data from the indexes that Graph Node builds.
 
-Before you start writing your own subgraph, you might want to have a look at the Graph Explorer and explore some of the subgraphs that have already been deployed. The page for each subgraph contains a playground that lets you query that subgraph's data with GraphQL.
+Before you start writing your own subgraph, you might want to have a look at the [Graph Explorer](https://thegraph.com/explorer) and explore some of the subgraphs that have already been deployed. The page for each subgraph contains a playground that lets you query that subgraph's data with GraphQL.
```

Comment:
```suggestion
Before you start writing your own subgraph, you might want to have a look at [Graph Explorer](https://thegraph.com/explorer) and explore some of the subgraphs that have already been deployed. The page for each subgraph contains a playground that lets you query that subgraph's data with GraphQL.
```

Source: https://www.notion.so/edgeandnode/The-Graph-Style-Guide-70a1409024d84bfbb4eccdf0e49b7aa7?pvs=4#4baa7d6f012a43bbadfc1d7180d9c47e (thanks @bermchain!)